src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1821,8 +1821,8 @@ class C
 
                 state.SendTypeChars("F")
                 Await state.AssertSelectedCompletionItem(displayText:="Fo", isHardSelected:=True)
-                state.SendTypeChars(".")
-                Assert.Contains("(Fo.", state.GetLineTextFromCaretPosition(), StringComparison.Ordinal)
+                state.SendTypeChars(":")
```

Comment:
Based on what @jcouv said, typing `.` here should commit "Fo()", right? Which means there's a bug in the provider somewhere?

Reply:
I have sync with Julien offline about this, and we agree that this dot behavior is OK